test/model/test/common/include/model/test/wiredtiger_util.h

```diff
@@ -218,49 +237,49 @@ wt_rollback_to_stable(WT_CONNECTION *conn)
         int __ret_model, __ret_wt;                                         \
         __ret_model = table->get_ext(key, __out_model, ##__VA_ARGS__);     \
         __ret_wt = wt_get_ext(session, uri, key, __out_wt, ##__VA_ARGS__); \
-        testutil_assert(__ret_model == __ret_wt);                          \
-        testutil_assert(__out_model == __out_wt);                          \
+        wt_model_assert_equal(__ret_model, __ret_wt);                      \
```

Comment:
If wt_model_assert_equal() fails in wt_model_assert() it will be difficult to tell which wt_mode_assert_equal() failed since both will reference the same line number. This could be fixed by adding a msg parameter to wt_mode_assert_equal() and printing that instead of Values.

Reply:
This actually works because the new macro uses # to stringify the arguments, so it will explicitly print `__ret_model` and `__ret_wt` to the console, or `__out_model` and `__out_wt`, together with their respective values.

Reply:
Here is an example of an output:
```
Assertion failure in void test_transaction_column_wt(): Values are not equal. 
  Location: /home/peter.macko/Projects/wt-12882-column-store-model/test/model/test/model_transaction/main.cpp:484
  __out_model: Value 3
  __out_wt: Value 4
```